src/extensions/default/bramble/lib/launcher.js

```diff
@@ -41,17 +41,15 @@ define(function (require, exports, module) {
         var server = _launcherInstance.server;
         var browser = _launcherInstance.browser;
 
+        callback = callback || function(){};
+
         server.serveLiveDocForUrl(url, function(err, urlOrHTML) {
             if(err) {
-                // TODO: how to deal with this error?                
                 console.error("[Launcher Error]", err);
-                return;
+                return callback();
```

Comment:
The callback is an invention of ours, and the Brackets callers of this don't pass one.  I don't want to leak errors back for no reason.  Also, there isn't much we can do with an error in this case.